components/Testimonial/index.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <div class="wrapper">
+    <div class="testimonial">
+      <img class="testimonial__image" width="358" height="497" src="~/assets/images/Testimonials.svg" alt="Testimonials">
+      <div class="testimonial__content">
+        <h3 class="testimonial__content-title">
+          Testimonial
+        </h3>
+        <h4 class="testimonial__content-subtitle">
+          Dari Para Pengguna Desa Digital
+        </h4>
+        <div class="mt-6 mb-8 sm:(mt-7) xl:(mb-10 grid grid-cols-2 items-end)">
+          <p class="testimonial__content-description">
+            Lorem ipsum dolor sit amet,
+            consectetur adipiscing elit. Enim aenean justo, adipiscing in lacinia mauris mauris vel.
+          </p>
+          <div class="mt-4 lg:(inline-flex items-baseline) xl:(inline)">
+            <p class="testimonial__content-text">
+              Testimoni dari
+            </p>
+            <BaseChipsGroup mandatory :values="listChips" @onChange="onClickChips" />
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+<script>
+export default {
+  data () {
+    return {
+      listChips: [
+        {
+          label: 'Semua',
+          value: 'semua'
+        },
+        {
+          label: 'Masyarakat',
+          value: 'masyarakat'
+        },
+        {
+          label: 'Mitra',
+          value: 'mitra'
+        }
+      ],
+      testimonials: [],
+      query: {
+        type: null
+      }
+    }
+  },
+  async fetch () {
+    const response = await this.$axios.get('/testimonials', { params: this.query })
+    const datas = response.data.data
```

Comment:
minor:

you can write this line like this:

```js
const { data } =  response.data

this.testimonials = data
```

because the word `data` is considered as plural

Reply:
I see kak, it's better to use the destructuring object.